Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -348,6 +348,10 @@ def _getMVASelectors(postfix):
     ptMin = 0,
 )
 
+# Select jets for JetCore tracking
+highPtJets = cms.EDFilter("CandPtrSelector", src = cms.InputTag("ak4CaloJets"), cut = cms.string("pt()>1000"))
+highPtJetsForTrk = cms.EDFilter("CandPtrSelector", src = cms.InputTag("ak4CaloJetsForTrk"), cut = cms.string("pt()>1000"))
```

Comment:
You could define this also as
```suggestion
highPtJetsForTrk = highPtJets.clone(src = "ak4CaloJetsForTrk")
```
so that the cut definition does not get repeated.

Reply:
ok done